source/elements/oneDPL/source/parallel_api/parallel_range_api.rst

```diff
@@ -0,0 +1,327 @@
+.. SPDX-FileCopyrightText: Contributors to the oneAPI Specification project.
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+Parallel Range Algorithms
+-------------------------
+
+oneDPL provides variations of algorithms that work with ranges defined in the `C++ Standard`_, 6th edition (C++20)
+and newer. These algorithms execute according to a oneDPL execution policy supplied as the first argument,
+similarly to other oneDPL algorithms.
+
+The oneDPL parallel range algorithms rely on the functionality of C++20 and are not available in the code
+compiled for earlier editions of the C++ standard.
+
+The parallel range algorithms reside in ``namespace oneapi::dpl::ranges``. Same as the range algorithm functions
+defined by the C++ standard in ``namespace std::ranges``, they cannot be found by argument-dependent name lookup
+and cannot be called with explicitly specified template arguments. [*Note*: A typical implementation uses
+predefined function objects which static function call operators have the required signatures. -- *end note*]
+
+The following differences to the standard C++ range algorithms apply:
+
+- Parallel range algorithms cannot be used in constant expressions.
```

Comment:
Do we want any mention here that arguments to parallel range algorithms must be sycl device copyable?

There is currently a discussion happening about pointer-to-member fields and functions and their device copyability.  It is currently not specified in the SYCL specification, and functionally has problems when the ABI is not the same between host and device.  With this being a somewhat common pattern in callables arguments to ranges APIs, we may want to mention something here at least about device copyability in general, if not about pointer-to-member fields specifically.

Reply:
Some general wording does make sense, but it is not really specific to parallel range algorithms, so better done in a separate patch.

The pointer-to-member issue seems more of an implementation problem (at least until the SYCL specification says otherwise), so making it a restriction here does not sound right.

Reply:
Yes, I agree that a general comment for oneDPL as a whole about device copyability with device policies is probably the correct approach.